apps/users/emails.py

```diff
@@ -85,3 +91,67 @@ def get_html_link(self, link_text, url):
         link = link_text.format('<a href="' + self.get_host() + url
                                 + '" target="_blank">', '</a>')
         return link
+
+
+class WelcomeEmail(SyncEmailMixin, EmailAplus):
+
+    def get_receivers(self):
+        receiver = self.object
+        return [receiver]
+
+    def get_overview_link(self):
+        url = reverse('userdashboard-overview')
+        link = self.get_host() + url
+        return link
+
+    def get_template_id(self, receiver):
+        language = self.get_receiver_language(receiver)
+        try:
+            template_id = int(settings.SENDINBLUE_TEMPLATES[language])
```

Comment:
why do we need the int here? and could it only be done at the return instead of in both conditions?

Reply:
We shouldn't need it, but I didn't want to rely too much on the settings. On the other hand, if that goes wrong, we will notice it and the int only works if someone adds the numbers as strings, so will just remove it and hope for the best. :p